lib/axon_onnx/deserialize.ex

```diff
@@ -1591,6 +1591,62 @@ defmodule AxonOnnx.Deserialize do
     {updated_axon, params, used_params}
   end
 
+  defp recur_nodes(
+         %Node{
+           op_type: "Resize",
+           input: input,
+           attribute: attrs,
+           output: output
+         },
+         {axon, params, used_params}
+       ) do
+
+    # op_type spec: https://github.com/onnx/onnx/blob/main/docs/Operators.md#resize
+
+    IO.inspect([input, output], label: "IO")
+    IO.inspect(options!(attrs), label: "attributes")
+
+    _roi = Map.get(params, "Resize_roi") |> IO.inspect(label: "roi") # [0.0, 0.0, 0.0, 0.0, 1.0, 1.0, 1.0, 1.0]
+    _sizes = Map.get(params, "Resize_sizes") |> IO.inspect(label: "sizes") # nil
+    scales = Map.get(params, "Resize_scales") |> IO.inspect(label: "scales") # [1.0, 1.0, 2.0, 2.0]
+
+    %{
+      "coordinate_transformation_mode" => _transformation,
+      "mode" => mode,
+      "nearest_mode" => "floor"
+    } = options!(attrs)
+
+    [in_layer_name | _ ] = input
```

Comment:
`constant!` asserts that an input must be a constant value known when we build the graph---it will check both the available Axon graphs and the input parameters

```suggestion
    [in_layer_name | rest_of_inputs] = input
    
    {_roi, scales, sizes} =
      case rest_of_inputs do
        [roi, scales] ->
          {roi, scales, nil}
        
        [roi, scales, sizes] ->
          {roi, scales, sizes}
      end
  
  # convert to integer type in Nx
  scales = constant!(scales, axon, params) |> Nx.as_type({:s, 64}) |> Nx.to_flat_list()
```

Reply:
according the spec `Only one of 'scales' and 'sizes' can be specified.`. I haven't thought of good way to differentiate those two yet. Do we have to look at the input names for that?

Reply:
Seems like the inputs are in order with missing items being empty strings, that makes the distinction a bit easier.